deploy/crds/integreatly_v1alpha1_smtpcredentialset_cr.yaml

```diff
@@ -1,13 +1,14 @@
 apiVersion: integreatly.org/v1alpha1
 kind: SMTPCredentialSet
 metadata:
+  # name must be between 1-40 characters
   name: example-smtpcredentialset
 spec:
-  # i want my blob storage information output in a Secret named test in the Namespace cloud-resource-operator
+  # i want my smtp cred information output in a Secret named test in the Namespace cloud-resource-operator
```

Comment:
same with all secret refs, I think we want the secret created in the product namespace?

Reply:
We do actually do that here: https://github.com/integr8ly/cloud-resource-operator/pull/3/files/a22b5b0546b2e1a574283d992c9ac51e984cc4b7#diff-1e673976e07ef62921f71dadba0ac6f4R148 

So I'll remove the `secretRef` attribute from the CRDs and auto-generate the secret name as well

Reply:
So this creates it in the same namespace as the resource is created in. I think this is fine but as long as we expect the resource to be created in the product namespace and as long as we limit the operator group to our product namespaces. We wouldn't want any one in the cluster to be able to create the resource.